components/Events/Filters.js

```diff
@@ -0,0 +1,228 @@
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+export default function filters(props) {
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [chosenComms, setChosenComms] = useState({
+    'studio': false,
+    'icpc': false,
+    'design': false,
+    'cyber': false,
+    'teach-la': false,
+    'w': false,
+    'ai': false,
+    'hack': false,
+    'board': false,
+  });
+  const [chosenDays, setChosenDays] = useState({
+    Sunday: false,
+    Monday: false,
+    Tuesday: false,
+    Wednesday: false,
+    Thursday: false,
+    Friday: false,
+    Saturday: false,
+  });
+  const [chosenTimes, setChosenTimes] = useState({
+    fromTime: '',
+    toTime: '',
+  });
+  const [chosenLoc, setChosenLoc] = useState({
+    'Online': false,
+    'In-Person': false,
+  });
+
+  const dateToMinutes = (date) => {
+    const itemDate = new Date(date);
+    const itemMins =
+      itemDate.getHours() * 60 + itemDate.getMinutes();
+    return itemMins;
+  };
+
+  const handleSearch = () => {
+    // Check search input
+    const lowerCaseSearch = searchValue.toLowerCase();
+    let allEvents = events.filter((item) =>
+      [item.title, item.committee, item.location, item.description].some(
+        (itemsToSearch) =>
+          String(itemsToSearch).toLowerCase().includes(lowerCaseSearch),
+      ),
+    );
+    // Check time filter
```

Comment:
nit: spacing between chunks for readability
```suggestion

    // Check time filter
```